src/components/FlyoutMenu/__snapshots__/FlyoutMenu.story.storyshot

```diff
@@ -72,103 +73,108 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT Exper
             role="button"
             tabIndex={0}
           />
-          <span
-            className="bx--visually-hidden"
-            role="link"
-            tabIndex="0"
-          >
-            Focus sentinel
-          </span>
           <div
-            className="bx--tooltip bx--tooltip--shown iot--flyout-menu--body iot--flyout-menu--body__bottom-start iot--flyout-menu--body__light iot--flyout-menu--body__open"
-            data-floating-menu-direction="bottom"
-            data-testid="flyout-menu"
-            disabled={false}
-            id="flyout-tooltip"
             onBlur={[Function]}
-            onContextMenu={[Function]}
-            onFocus={[Function]}
-            onMouseOut={[Function]}
-            onMouseOver={[Function]}
-            open={true}
-            role="tooltip"
-            style={
-              Object {
-                "margin": 0,
-                "opacity": 1,
-                "position": "absolute",
-                "top": "0px",
-                "visibility": "hidden",
-              }
-            }
           >
             <span
-              className="bx--tooltip__caret"
-            />
+              className="bx--visually-hidden"
+              role="link"
+              tabIndex="0"
+            >
+              Focus sentinel
+            </span>
             <div
-              aria-labelledby="test-trigger-id-2"
-              className="bx--tooltip__content"
-              role="dialog"
-              tabIndex="-1"
+              className="bx--tooltip bx--tooltip--shown iot--flyout-menu--body iot--flyout-menu--body__bottom-start iot--flyout-menu--body__light iot--flyout-menu--body__open"
+              data-floating-menu-direction="bottom"
+              data-testid="flyout-menu"
+              disabled={false}
+              id="flyout-tooltip"
+              onBlur={[Function]}
+              onContextMenu={[Function]}
+              onFocus={[Function]}
+              onMouseOut={[Function]}
+              onMouseOver={[Function]}
+              open={true}
+              role="tooltip"
+              style={
+                Object {
```

Comment:
should this be inline?

Reply:
Yep, it's how Carbon does the tooltip positioning/offset